backend/courses/util.py

```diff
@@ -495,8 +497,12 @@ def upsert_course_from_opendata(info, semester, missing_sections=None):
     set_instructors(section, info["instructors"])
     add_associated_sections(section, info["linked_courses"])
 
-    # add_attributes(section, info["attributes"])  # TODO: save attributes (course or section?)
-    # add_restrictions(section, info["restrictions"])  # TODO: add registration restrictions
+    # delete all attributes and restrictions
+    Attribute.objects.all().delete()
```

Comment:
Oh that's a great point. I'll remove the `delete()`s. I don't think there's a point in doing the clear either, since every attribute (restriction) would historically be associated with some section.